core/components/atoms/segmentedControl/SegmentedControl.tsx

```diff
@@ -0,0 +1,338 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { BaseProps, extractBaseProps } from '@/utils/types';
+import {
+  SegmentedControlContext,
+  SegmentedControlContextValue,
+  SegmentedControlValue,
+  SegmentedControlSize,
+} from './SegmentedControlContext';
+import { SegmentedControlItem, SegmentedControlItemProps } from './SegmentedControlItem';
+import styles from '@css/components/segmentedControl.module.css';
+
+export type { SegmentedControlValue, SegmentedControlSize };
+
+export interface SegmentedControlProps extends BaseProps {
+  /**
+   * Index of desired selected segment.
+   */
+  activeIndex?: number;
```

Comment:
Please define in prop table whether the indexing starts from 0 or 1,  it might be confusing for some user